### ISSUE TITLE
When a smoke alarm request is made, process the latitude and longitude of the requestor's address so that it can be pushed to allReady

### ISSUE THREAD

**Issue by OhMcGoo** [redcross/smoke-alarm-portal#206]
When a smoke alarm request is made, process (and store) the latitude and longitude of the requestor's address so that it can be pushed to allReady along with rest of the request data.

**Comment by cecilia-donnelly** [comment]
Hey, @OhMcGoo.  We could manage this by taking the address we  receive from the user, sending it out to a mapping service (e.g. the Google Maps API), getting the lat/long back, and then including that info in the request we send to AllReady.  

If, however, AllReady is already using some kind of mapping service then maybe we should just send the address to them.  There's no particular advantage or disadvantage to doing the mapping on our end or theirs as far as I know.  @MisterJames, it sounded on the last standup like someone was working on mapping capabilities in AllReady.  Is that right?  How does AllReady currently handle finding latitudes and longitudes, if it does at all?

**Comment by OhMcGoo** [comment]
@cecilia-donnelly, I think we should do this on the **getasmokealarm** side. Here's why: We don't know if everyone will choose to use **allReady**. We do know that everyone (except Kansas-Nebraska) is using getasmokealarm. Thus, I think it would be most useful to calculate lats and longs before the data gets pushed to allReady. Better, would be a way of validating that the address entered is _real_. I looked at all of the smoke alarm requests over the weekend and a lot of the entries are pretty messy or simply unintelligible.

**Comment by cecilia-donnelly** [comment]
@OhMcGoo, makes sense.  Let's prioritize this with @kfogel.  I think this could be a great task for GiveCamp.
